src/routes/v1/mod.rs

```diff
@@ -9,6 +11,23 @@ mod versions;
 pub fn v1_config(cfg: &mut web::ServiceConfig) {
     cfg.service(
         web::scope("api/v1")
+            .wrap_fn(|req, srv| {
+                let current_minute = Utc::now().timestamp() / 60;
+
+                if current_minute % 10 > 5 {
```

Comment:
Hmm, this seems too frequent. I was more thinking 5-10 minutes per hour, and then later 6-12 hours at a time. Do you think this approach is better, though?

Reply:
I did this bc it was easiest to code, but if you want you can commit to this branch and change it. I think either works but more frequent is probably better IMO (most applications using API V1 would already have seen the messages appended to every data field)

Reply:
Yeah that's fair